client/src/Component/SignIn/SignIn.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import './SignIn.css';
+import { useMediaQuery } from 'react-responsive';
+import Grid from '@material-ui/core/Grid';
+import {toast} from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+toast.configure();
+
+const SignUp = ()=>{
```

Comment:
Change SignUp to SignIn, you copied the code of #42. Don't do this next time.

Reply:
Yup everything was same so i just modified the code as per my need but i forget the change the main thing. My fault and sorry for that. Will keep that in mind.